packages/typescript/src/internal/devserver/ts_devserver.bzl

```diff
@@ -138,11 +145,25 @@ ts_devserver = rule(
         ),
         "devserver": attr.label(
             doc = """Go based devserver executable.
+
+            With cross-platform RBE for OSX & Windows ctx.executable.devserver will be linux as --cpu and
+            --host_cpu must be overridden to k8. However, we still want to be able to run the devserver on the host
+            machine so we need to include the host devserver binary, which is ctx.executable.devserver_host, in the
+            runfiles. For non-RBE and for RBE with a linux host, ctx.executable.devserver & ctx.executable.devserver_host
+            will be the same binary.
+
             Defaults to precompiled go binary in @npm_bazel_typescript setup by @bazel/typescript npm package""",
             default = Label("//devserver"),
             executable = True,
             cfg = "host",
         ),
+        "devserver_host": attr.label(
+            doc = """Go based devserver executable for the host platform.
+            Defaults to precompiled go binary in @npm_bazel_typescript setup by @bazel/typescript npm package""",
+            default = Label("//devserver:devserver_%s" % host_platform),
```

Comment:
is there some other way we could get this info so we don't need to create the new index.bzl file and make the rollout complicated? @meteorcloudy maybe you have some idea - I know there's a starlark trick to know if we are on Windows or not